### PR TITLE
Nbo/navigate basic workspace selector

### PR DESCRIPTION
**Pull request by nborde-CSM**
Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
             "launch": "Update and launch scenario"
           }
         }
-      }
+      }
```
Comment (csm-thu) [inline public/locales/en/translation.json:48]: Trailing spaces to remove here :)

File: public/locales/fr/translation.json [hunk 2]
```diff
@@ -450,7 +450,23 @@
...
+        "button": {
+          "open": "Ouvrir"
+        }
+      },
+      "placeholder": {
+        "noworkspace": {
+          "title": "Aucun workspace n'est accessible",
+          "description": {
+            "reason": "Ceci peut signifier que vous n'avez aucune permission pour aucun workspace existant.",
+            "contact": "Si vous pensez qu'il s'agit d'une erreur, veillez contacter votre administrateur."
```
Comment (csm-thu) [inline public/locales/fr/translation.json:465]: Typo: `veillez` -> `veuillez`

File: public/locales/fr/translation.json [hunk 3]
```diff
@@ -450,7 +450,23 @@
...
+      "card": {
+        "button": {
+          "open": "Ouvrir"
+        }
+      },
+      "placeholder": {
+        "noworkspace": {
+          "title": "Aucun workspace n'est accessible",
+          "description": {
+            "reason": "Ceci peut signifier que vous n'avez aucune permission pour aucun workspace existant.",
```
Comment (csm-thu) [inline public/locales/fr/translation.json:464]: La tournure "vous n'avez aucune permission pour aucun workspace existant" est un peu étrange, non ? On pourrait peut-être remplacer ça par "vous n'avez les permissions nécessaires pour accéder aux workspaces existants" ?

File: src/state/hooks/OrganizationHooks.js [hunk 4]
```diff
@@ -14,3 +14,7 @@ export const useOrganizationData = () => {
 export const useOrganizationId = () => {
   return useSelector((state) => state.organization.current?.data?.id);
 };
+
+export const useOrganisationName = () => {
```
Comment (csm-thu) [inline src/state/hooks/OrganizationHooks.js:18]: Typo: `Organisation` -> `Organization`

File: src/views/Workspaces/Workspaces.js [hunk 5]
```diff
@@ -3,13 +3,87 @@
...
 const Workspaces = () => {
-  // const navigate = useNavigate();
+  const navigate = useNavigate();
+  const { t } = useTranslation();
+  const { workspacesList, organizationName, selectWorkspace } = useWorkspaces();
+  sessionStorage.removeItem('providedUrlBeforeSignIn');
+
+  const labels = {
+    openButtonWorkspaceCard: t('genericcomponent.workspaceselector.card.button.open', 'Open'),
+    titleNoWorkspacePacePlaceHolder: t(
```
Comment (csm-thu) [inline src/views/Workspaces/Workspaces.js:20]: Typo: `WorkspacePace` , it appears several times in this file

File: cypress/commons/constants/generic/TestConstants.js [hunk 6]
```diff
@@ -8,6 +8,7 @@ export const AUTH_QUERY_URL =
   'https://login.microsoftonline.com/e413b834-8be8-4822-a370-be619545cb49/oauth2/v2.0/token';
 export const LOCAL_WEBAPP_URL = 'http://localhost:3000';
 export const URL_POWERBI = `${LOCAL_WEBAPP_URL}/api/get-embed-info`;
+export const DEFAULT_WORKSPACE_ID = 'W-rXeBwRa0PM';
```
Comment (csm-thu) [inline cypress/commons/constants/generic/TestConstants.js:11]: Could we rename this constant to `BREWERY_WORKSPACE_ID` instead ? We're already using "default workspace" in the fixtures for a stubbed workspace with id "W-stbbdbrwry"

File: cypress/integration/brewery/Router_RedirectionAfterLogin.spec.js [hunk 7]
```diff
@@ -13,9 +13,7 @@ describe('redirection after login', () => {
       GET_SOLUTIONS: true,
     });
   });
-  beforeEach(() => {
-    stub.setFakeWorkspaceId('W-stbbdbrwry');
-  });
+  beforeEach(() => {});
```
Comment (csm-thu) [inline cypress/integration/brewery/Router_RedirectionAfterLogin.spec.js:16]: `beforeEach` can probably be removed if it si empty :slightly_smiling_face:

File: cypress/fixtures/stubbing/default/workspaces.js [hunk 8]
```diff
@@ -19,8 +29,95 @@ export const WORKSPACE_EXAMPLE = {
...
   useDedicatedEventHubNamespace: true,
   sendScenarioMetadataToEventHub: true,
-  security: { default: 'admin', accessControlList: [] },
+  security: { default: ROLES.SCENARIO.ADMIN, accessControlList: usersAccess },
 };
 
 export const DEFAULT_WORKSPACE = WORKSPACE_EXAMPLE;
-export const DEFAULT_WORKSPACES_LIST = [DEFAULT_WORKSPACE];
+
+export const DEFAULT_WORKSPACES_LIST = [
```
Comment (csm-thu) [inline cypress/fixtures/stubbing/default/workspaces.js:37]: Do we really need 12 workspaces ?
Also, the `DEFAULT_WORKSPACES_LIST` should be the most simple case to run most of our tests, it should probably be a list with only one workspace in it.

File: cypress/integration/workspaces/SelectWorkspaces.spec.js [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+      Workspaces.getWorkspaceCardById(workspace.id).should('be.visible');
+    });
+    Workspaces.selectWorkspace(stub.getWorkspaces()[0].id);
+    Scenarios.getScenarioViewTab(60).should('be.visible');
+
+    Scenarios.getScenarioView();
+    Workspaces.getHomeButton().should('be.visible').click();
+  });
+
+  it('Check routing behavior whith no workspace available', () => {
```
Comment (csm-thu) [inline cypress/integration/workspaces/SelectWorkspaces.spec.js:31]: Typo: `whith` -> `with`

File: cypress/fixtures/stubbing/default/workspaces.js (hunk 8 above)
Comment (nborde-CSM) [reply cypress/fixtures/stubbing/default/workspaces.js:37]: The purpose of the `DEFAULT_WORKSPACES_LIST` is also to test the graphic features of the workspace selector. So, it should be convenient to test an "extreme" case.

File: cypress/integration/workspaces/SelectWorkspaces.spec.js [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+    stub.start();
+  });
+
+  after(() => {
+    stub.stop();
+  });
+
+  it('Can select one workspace then return to workspace selector by home button', () => {
+    Login.loginWihtoutWorkspace();
+    Workspaces.getWorkspacesView();
```
Comment (csm-thu) [inline cypress/integration/workspaces/SelectWorkspaces.spec.js:20]: You're getting an element but do nothing with it, maybe you should add a simple exist/visible check ?

Comment (nborde-CSM) [reply cypress/integration/workspaces/SelectWorkspaces.spec.js:20]: It's a mess. I should have test if it is visible.

File: cypress/integration/workspaces/SelectWorkspaces.spec.js [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+    stub.setWorkspaces([]);
+    Login.loginWihtoutWorkspace();
+    Workspaces.getNoWorkspacePlaceHolder().should('be.visible');
+  });
+
+  it('Check routing behavior whith only one workspace available', () => {
+    stub.setWorkspaces([WORKSPACE_EXAMPLE]);
+    Login.loginWihtoutWorkspace();
+
+    Workspaces.getHomeButton().should('not.exist');
```
Comment (csm-thu) [inline cypress/integration/workspaces/SelectWorkspaces.spec.js:41]: We should also check that the scenario view is displayed

File: src/views/Workspaces/Workspaces.js (hunk 5 above)
Comment (esasova) [reply src/views/Workspaces/Workspaces.js]: And placeHolder one more time (same on lines 24 and 28) :slightly_smiling_face:

File: cypress/commons/actions/generic/Login.js [hunk 12]
```diff
@@ -23,26 +33,44 @@ function login(url = BASE_URL) {
...
+
   getMicrosoftLoginButton().click();
   api.waitAlias(reqAuthAlias);
   api.waitAliases(newPageQueries, { timeout: 60 * 1000 });
   Scenarios.getScenarioViewTab(60).should('be.visible');
   cy.saveLocalStorage();
 }
 
-function relogin(url = BASE_URL) {
+function loginWihtoutWorkspace() {
```
Comment (esasova) [inline cypress/commons/actions/generic/Login.js:54]: typo: wiTHout (same at line 82) :slightly_smiling_face:

File: cypress/fixtures/stubbing/default/workspaces.js [hunk 13]
```diff
@@ -19,8 +29,95 @@ export const WORKSPACE_EXAMPLE = {
   sendInputToDataWarehouse: true,
   useDedicatedEventHubNamespace: true,
   sendScenarioMetadataToEventHub: true,
-  security: { default: 'admin', accessControlList: [] },
+  security: { default: ROLES.SCENARIO.ADMIN, accessControlList: usersAccess },
 };
 
 export const DEFAULT_WORKSPACE = WORKSPACE_EXAMPLE;
```
Comment (esasova) [inline cypress/fixtures/stubbing/default/workspaces.js:35]: Isn't it redundant to assign an existing const to a new const ? :thinking:

File: cypress/fixtures/stubbing/default/workspaces.js (hunk 8 above)
Comment (nborde-CSM) [reply cypress/fixtures/stubbing/default/workspaces.js:37]: So, I will create an EXTENTED_WOKSPACES_LIST

File: src/components/AppBar/components/HomeButton.js [hunk 14]
```diff
@@ -22,9 +22,9 @@ export const HomeButton = () => {
     <Tooltip
```
[the remark this answers is not in the document]
Comment (csm-thu) [reply src/components/AppBar/components/HomeButton.js:22]: +1 
@nborde-CSM you may have to rebase the **_feat/multi_workspace_** branch on **_main_** in the UI components repo

File: cypress/fixtures/stubbing/default/workspaces.js (hunk 13 above)
Comment (nborde-CSM) [reply cypress/fixtures/stubbing/default/workspaces.js:27]: It was already like that. I don't modify it to avoid breaking some things.

Comment (esasova) [reply cypress/fixtures/stubbing/default/workspaces.js:27]: You are right ! thanks :pray:

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Pull request closed without merging**